edit-post/assets/stylesheets/_variables.scss

```diff
@@ -56,6 +56,8 @@ $block-parent-side-ui-clearance: $parent-block-padding - $block-padding; // spac
 
 $block-container-side-padding: $block-side-ui-width + $block-padding + 2 * $block-side-ui-clearance;
 
+$default-block-margin: $block-padding * 2 + $block-spacing; // This is the default margin between blocks.
```

Comment:
Minor: We should use parentheses so the future maintainer doesn't need to recall their [order of operations](https://en.wikipedia.org/wiki/Order_of_operations) 😄